### PR TITLE
Fix ModuleFactory::Find_Module_Interface_Mask function name

### PR DESCRIPTION
**Pull request by xezon**
Shows as **findModuleInterfaceMask** in Decompiler. So I adjusted name accordingly.

### CODE REVIEW

File: src/game/common/thing/thingtemplate.cpp [hunk 1]
```diff
@@ -474,9 +474,9 @@ void ThingTemplate::Init_For_LTA(const Utf8String &name)
     strncpy(buffer, name, ARRAY_SIZE(buffer));
     int i;
 
-    for (i = 0; buffer[i]; i++) {
+    for (i = 0; buffer[i]; ++i) {
```
Comment (jonwil) [inline src/game/common/thing/thingtemplate.cpp:477]: Why make this unrelated change?

File: src/game/common/thing/thingtemplate.cpp [hunk 2]
```diff
@@ -474,9 +474,9 @@ void ThingTemplate::Init_For_LTA(const Utf8String &name)
...
         if (buffer[i] == '/') {
-            i++;
+            ++i;
```
Comment (jonwil) [inline src/game/common/thing/thingtemplate.cpp:479]: Why make this unrelated change?

**Review by jonwil — CHANGES_REQUESTED** [review]
Other than the unnecessary unrelated changes, this looks OK to me.

File: src/game/common/thing/thingtemplate.cpp (hunk 1 above)
Comment (xezon) [reply src/game/common/thing/thingtemplate.cpp:477]: Decompiler shows like so. Plus Pre-increment is always the default that should be used unless post-increment is explicitly required.

**Review by jonwil — APPROVED** [review]
Looks good to me.

**Review by tomsons26 — APPROVED** [review]
